src/clims/services/container.py

```diff
@@ -158,6 +159,14 @@ def __setitem__(self, key, value):
         # the container is saved
         self._locatables[ix.raw] = value
 
+    @property
+    def contents(self):
+        c = []
```

Comment:
FYI: I've also added a `contents` attribute to the container - but it's in my next PR. I had it return only the items that have something in it, while iterating through the container itself returns all items (even the empty ones). I'll synch with you when I merge the two.

Reply:
Ok!